### DISCUSSION TITLE
e-puck speeds conflict

### DISCUSSION THREAD

**Discussion by DisaacD** [cyberbotics/webots#6444] [General]
Hi,
According to https://cyberbotics.com/doc/guide/epuck?version=R2021b, e-puck robot has the following parameters:
Wheel radius: 20.5 mm
Max. forward/backward speed: 0.25 m/s
Max. rotation speed: 6.28 rad/s

Why the max fwd /bck speed is not 6.28*20.5 mm/sec?
Cheers

**Comment by omichel** [comment]
You are right, this is a mistake. I recomputed the actual speed from the actual data in the PROTO file: [wheel radius is 20 mm](https://github.com/cyberbotics/webots/blob/49783d897a13f54273f47dcdead65eecef4376ee/projects/robots/gctronic/e-puck/protos/E-puck.proto#L193) and [maximum rotation speed is 7.536](https://github.com/cyberbotics/webots/blob/49783d897a13f54273f47dcdead65eecef4376ee/projects/robots/gctronic/e-puck/protos/E-puck.proto#L61). Therefore the maximum forward/backward speed is 0.15 m/s.
I updated in this #6447. Can you please review it and approve it if you agree?

**Reply by DisaacD** [reply]
Thank you for the prompt response.
